### PR TITLE
Russian(RU) numeral and ordinal improvements

### PR DESCRIPTION
**Pull request by evjava**
- added non-typo variant for 11 (одиннадцать)
- added variants for grammatical cases

### CODE REVIEW

File: Duckling/Ordinal/RU/Rules.hs [hunk 1]
```diff
@@ -39,6 +39,7 @@ ordinalsFirstthMap = HashMap.fromList
   , ( "девят", 9 )
   , ( "десят", 10 )
   , ( "одинадцат", 11 )
```
Comment (michniewicz) [inline Duckling/Ordinal/RU/Rules.hs]: Can `одинадцат` be safely removed from the list then? It appears to be grammatically incorrect.
Ref: https://en.wiktionary.org/wiki/%D0%BE%D0%B4%D0%B8%D0%BD%D0%BD%D0%B0%D0%B4%D1%86%D0%B0%D1%82%D1%8C

File: Duckling/Ordinal/RU/Rules.hs [hunk 2]
```diff
@@ -66,7 +67,7 @@ ruleOrdinalsFirstth :: Rule
 ruleOrdinalsFirstth = Rule
   { name = "ordinals (first..19th)"
   , pattern =
-    [ regex "(перв|втор|трет|четверт|пят|шест|седьм|восьм|девят|десят|одинадцат|двенадцат|тринадцат|четырнадцат|пятнадцат|шестнадцат|семнадцат|восемнадцат|девятнадцат|двадцат)(ье(го|й)?|ого|ый|ой|ий|ая|ое|ья)"
+    [ regex "(перв|втор|трет|четверт|пят|шест|седьм|восьм|девят|десят|одинадцат|одиннадцат|двенадцат|тринадцат|четырнадцат|пятнадцат|шестнадцат|семнадцат|восемнадцат|девятнадцат|двадцат)(ье(го|й)?|ого|ый|ой|ий|ая|ое|ья)"
```
Comment (michniewicz) [inline Duckling/Ordinal/RU/Rules.hs]: ditto

File: Duckling/Ordinal/RU/Rules.hs (hunk 1 above)
Comment (evjava) [reply Duckling/Ordinal/RU/Rules.hs]: Fixed.

File: Duckling/Ordinal/RU/Rules.hs (hunk 2 above)
Comment (evjava) [reply Duckling/Ordinal/RU/Rules.hs]: Fixed.

**Comment by chessai** [comment]
Looks like these changes already made it into master via another PR(s). Thanks!

**Pull request closed without merging**